charts/steward/crds/tenants.yaml

```diff
@@ -9,6 +9,9 @@ spec:
     kind: Tenant
     singular: tenant
     plural: tenants
+    shortNames:
+    - stn
```

Comment:
`st` seems to be free as well. But chances are higher to conflict with future official resources.